source/plugin/plugin-class.rst

```diff
@@ -62,28 +62,39 @@ interacting with the game, such as registering commands or events.
 
 Your plugin can listen for particular events, called **state events**, to be notified about changes in the state of the
 game. In the example below, ``onServerStart()`` is called when the :javadoc:`GameStartedServerEvent` occurs; take note
-of the :javadoc:`Listener` annotation before the method.
+of the :javadoc:`Listener` annotation before the method.  
+
+The example below will log a message upon starting the server.  If your plugin is correctly loaded,
```

Comment:
Still two spaces before If in this sentence, but that has no effect on the preview itself. Everything else is fine.